File: ToyInterpreter 2/src/Model/IOutBuffer.java
package Model;

import mobserver.OutBufferObserver;
import mobserver.VarTableObserver;

public interface IOutBuffer {
	public void add(String s);
	public int getSize();
	public void printBuffer();
	public String toString();
	public void addObserver(OutBufferObserver o);
}

File: ToyInterpreter/ToyInterpreter/Model/IOutBuffer.cs
using System;
using ToyInterpreter.Model.MObserver;

namespace ToyInterpreter.Model
{
	public interface IOutBuffer
	{
		void add(String s);
		int getSize();
		void printBuffer();
		String toString();
		void addObserver(OutBufferObserver o);
	}
}

